tests/validator.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {edvConfig} from './ConfigSchema';
+import Ajv from 'ajv';
+
+const ajv = new Ajv({verbose: true, removeAdditional: false});
+ajv.addSchema(edvConfig, 'edvConfig');
+
+// throws if validation fails
+export function validateSchema({payload}) {
+  // validate payload against JSON schema
+  const valid = ajv.validate('edvConfig', payload);
+  if(valid) {
+    return true;
+  }
+  const error = new SyntaxError('Validation error.');
```

Comment:
Is `SyntaxError` the right choice here?  I think the specs say `SyntaxError` is for JavaScript parsing errors.  It's in test code so not a big deal.  Hard to map the few core errors to custom ones, but maybe just `Error` is better in this case?